build.sbt

```diff
@@ -101,10 +101,11 @@ lazy val dumpEventSchema = (project in file("sbt-dump-event-schema"))
     publishM2Configuration := publishM2Configuration.value.withResolverName(localMavenResolverForSbtPlugins.name),
     scriptedLaunchOpts := {
       scriptedLaunchOpts.value ++
-      Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
+        Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
     },
     scriptedBufferLog := false)
 
+//Warning: when publishing artefacts two times, without clean in between, sometimes instead of jar tar, plugin is published as normal jar
```

Comment:
Maybe open an issue and reference and here and in config.yml

Also, for:
1. easy searching of places where the given issue is referenced
2. to make https://virtuslab.github.io/tips/#jetbrains-products/linking-issues-in-jb-ides possible
3. to prevent dangling TODOx/FIXMEs with no associated github issue,

pls add a script like this one: https://github.com/VirtusLab/git-machete-intellij-plugin/blob/develop/scripts/enforce-issue-number-for-todos and run from the CI

Reply:
(can be on a separate PR ofc)